extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -121,7 +126,16 @@ export class ArticleComponent {
           </div>`;
 
       const {image} = htmlRefs(imgEl);
-      addAttributesToElement(image, dict({'src': articleData.image}));
+
+      if (isServedFromCache(doc) && !isAbsoluteUrl(articleData.image)) {
+        const fullUrl = doc.location.href + articleData.image;
```

Comment:
This won't work with all relative URLs, eg: `../../assets/image.jpg`

https://github.com/ampproject/amphtml/blob/master/src/url.js#L569

Reply:
Good catch, changed it and added that case to the unit tests.